index.d.ts

```diff
@@ -36,4 +36,4 @@ export interface playwrightLighthouseConfig {
  */
 export function playAudit(
   playwrightLHConfiguration: playwrightLighthouseConfig
-): Promise<void>;
+): Promise<any>;
```

Comment:
Can we construct the object type or specifically obtain the returned object type for the lighthouse itself?

Reply:
I wanted to, but I saw that there are no exposed types in Lighthouse library:
https://github.com/GoogleChrome/lighthouse/issues/1773